pages/glossary/automatic-error-detection.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Automatic detection of input errors
+key: automatic-error-detection
+---
+
+Automatic detection of input errors is the result of automatically carried procedures to verify that the provided input meets the requirements. 
```

Comment:
This doesn't actually describe what an "automatically detected input error" is. You're saying it's the result of something, but not what that thing is.

Reply:
Reworded